src/AlgebraicGeometry/Schemes/AffineAlgebraicSet/Objects/Methods.jl

```diff
@@ -26,7 +26,7 @@ end
 function Base.show(io::IO, X::AffineAlgebraicSet{<:Field,<:MPolyQuoRing})
   io = pretty(io)
   if get(io, :supercompact, false)
-    print(io, "Scheme")
+    print(io, "Affine algebraic set")
```

Comment:
The "detailed show" method below calls this "Reduced subscheme"

Reply:
The idea of the supercompact printing was to just give the essential, and no dependence on the type parameters. All of them are affine algebraic set: some are given by an ideal, others are not and we "just know" that they are the reduced subscheme of a given object. So for the non supercompact printings, we show what we know. The problem is that for objects of the same type (maybe different type parameters), we do not always have the same amount of information...

Reply:
I just didn't realize that the "detailed printing" *below* this method is not the matching one; the matching one is *above*, and it is consistent. So that is fine!